### PR TITLE
Use AsCommand in ValidateConnectionsCommand and ...

### PR DESCRIPTION
**Pull request by ehymel**
Fix symfony6 deprecation... 
Use AsCommand in ValidateConnectionsCommand and remove changes to $defaultName and $defaultDescription

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "secit-pl/imap-bundle",
```
Comment (Chris53897) [inline composer.json:2]: I think this must be reverted.

**Comment by Chris53897** [comment]
This can only me merged, if support for symfony 4.4 is dropped. Minimum 5.3. That was the reason i outcommented it, in the first place. But symfony 4.4 is no longer LTS, so i am in favour of dropping support.

**Comment by ThomasLandauer** [comment]
Depends on https://github.com/secit-pl/imap-bundle/pull/34

**Comment by secit-pl** [comment]
The new bundle version 3.0.0 was released which fixed deprecation. Please migrate to 3.0.0.

Caution: please read the migration guide at the end of the documentation before migrating from version 2.x to 3.x.

**Pull request closed without merging**